crates/node/p2p/src/macros.rs

```diff
@@ -0,0 +1,10 @@
+//! Macros for recording metrics.
+
+/// Sets a metric value.
+#[macro_export]
+macro_rules! set {
```

Comment:
```suggestion
macro_rules! record_metric {
```
a more helpful name would be nice